src/clr-core/button/button.stories.ts

```diff
@@ -84,120 +84,143 @@ export const API = () => {
         @click=${action('click')}>
         ${size === 'icon' ? html`<cds-icon></cds-icon>` : slot}
       </cds-button>
-        </cds-demo>
+    </cds-demo>
   `;
 };
 
 export const form = () => {
   return html`
-    <form @submit="${(e: Event) => {
+    <form cds-layout="vertical gap:sm" @submit="${(e: Event) => {
       e.preventDefault();
       action('submit')(e);
     }}">
-      <label for="name">Name</label><br />
-      <input id="name" /><br />
+      <div cds-layout="vertical gap:xs">
+        <label for="name" cds-text="caption">Name</label>
+        <input id="name" />
+      </div>
       <cds-button type="submit">submit</cds-button>
     </form>
   `;
 };
 
 export const actions = () => {
   return html`
-    <cds-button>solid</cds-button>
-    <cds-button action="outline">outline</cds-button>
-    <cds-button action="flat">link</cds-button>
+    <div cds-layout="horizontal gap:xs">
+      <cds-button>solid</cds-button>
+      <cds-button action="outline">outline</cds-button>
+      <cds-button action="flat">link</cds-button>
+    </div>
   `;
 };
```

Comment:
This is more of a fleeting thought, but having layout code in the component demos is there any concern of confusion for the devs copy/pasting it since this is the same code that renders in the docs? I was kind of on the fence on it so I wasnt sure if anyone had an opinion.

Reply:
I think, if the component that is being demoed is commonly expected to be in a layout container (e.g. form with a bunch form fields?), it's ok to leave it there because it's beneficial for the devs to copy/paste it including the layout.

If the button demos could render reasonably without it maybe it's better to not include the layout code? What do y'all think?

Reply:
Looking at the button demo though, I can see it's pretty useful here to space out the buttons that are grouped together. So I'm really ok with leaving it as is too..

Reply:
It was an easy way to dogfood layouts and solved the disappearing margins due to how browsers were handling "whitespace" between the buttons. Per our meeting today, we'll keep it in.